examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -5,12 +5,21 @@
 
 Images are represented by discrete pixels, either on the screen or in an
 image file.  When data that makes up the image has a different resolution
-than its representation on the screen we will see aliasing effects.
+than its representation on the screen we will see aliasing effects.  How
+noticeable these are depends on how much down-sampling takes place in
+the change of resolution (if any)
```

Comment:
```suggestion
the change of resolution (if any.)
```

Reply:
fix, but I believe th period belongs outside the parentheses according to the first two hits on google "parentheses at end of sentence"